winui/DataGrid/Row-Drag-and-Drop.md

```diff
@@ -0,0 +1,217 @@
+* `DraggingRows` : Gets the records which contains the data associated while dragging the rows. 
+* `DropPosition` : Gets a value indicating the drop position which is based on dropped location 
+* `TargetIndex` : Gets a value that indicates the target index where the record is going to be drop.
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.RowDropping += SfDataGrid_RowDropping;
+
+private void SfDataGrid_RowDropping(object sender, DataGridRowDroppingEventArgs e)
+{
+
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Row Dropped event
+
+`RowDropped` event occurs after dropped the record. The `DataGridRowDroppedEventArgs` has the following members, which provide information for the `Dropped` event.
+* `Data` : Gets a data object that contains the data associated while dragging the rows. 
+* `DraggingRows` : Gets the records which contains the data associated while dragging the rows.
+* `DropPosition` : Gets a value indicating the drop position which is based on dropped location 
+* `TargetIndex` : Gets a value that indicates the target index where the record is going to be drop.
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.RowDropped += SfDataGrid_RowDropped;
+
+private void SfDataGrid_RowDropped(object sender, DataGridRowDroppedEventArgs e)
+{
+    
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing row drag and drop operation
+
+### Disable dragging of certain rows in WinUI DataGrid
+
+You can restrict the dragging of certain rows in SfDataGrid by using the  `SfDataGrid.RowDragStarting` event.
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.RowDragStarting += SfDataGrid_RowDragStarting;
+
+private void SfDataGrid_RowDragStarting(object sender, DataGridRowDragStartingEventArgs e)
+{
+    var records = e.DraggingRows;
+    var orders = records[0] as OrderInfo;
+    if (orders.OrderID > 1005)
+        e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Disable dropping over certain rows in WinUI DataGrid
+
+You can restrict the dropping the records in certain rows in SfDataGrid by using the `SfDataGrid.RowDropping` event.
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.RowDropping += SfDataGrid_RowDropping;
+
+private void SfDataGrid_RowDropping(object sender, DataGridRowDroppingEventArgs e)
+{
+    if(e.TargetIndex > 6)
+        e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Disable the drag UI
+
+You can disable the draggable popup by setting the `ShowDragUI` as `false` in `SfDataGrid.RowDragOver` event.
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.RowDragOver += SfDataGrid_RowDragOver;
+
+private void SfDataGrid_RowDragOver(object sender, DataGridRowDragOverEventArgs e)
+{
+    e.ShowDragUI = false;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Reorder the source collection while drag and drop the rows
+
+You can reorder the source collection after drag and drop the row by handling `SfDataGrid.RowDropped` event.
```

Comment:
You can reorder the source collection after dragging and dropping the row by handling the `SfDataGrid.RowDropped` event.